crates/bitwarden-vault/src/cipher/cipher.rs

```diff
@@ -125,7 +125,17 @@ pub struct CipherView {
     pub revision_date: DateTime<Utc>,
 }
 
-#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+#[derive(Serialize, Deserialize, Debug, JsonSchema, PartialEq)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Enum))]
+pub enum CipherListViewType {
+    Login { fido2: bool },
```

Comment:
Would this be better with a more descriptive name, like `has_fido2` or `has_fido2_credentials`